constraint_utils/generic.py

```diff
@@ -160,6 +170,12 @@ def rebin_methylation(t: Union[hl.MatrixTable, hl.Table], bins: int=20) -> Union
 
 
 def trimer_from_heptamer(t: Union[hl.MatrixTable, hl.Table]) -> Union[hl.MatrixTable, hl.Table]:
+    """
+    Trim heptamer context to create trimer context.
+
+    param: t: Input Table with context annotation.
```

Comment:
```suggestion
    :param t: Input MatrixTable or Table with context annotation.
```
sorry if I added typos in my last suggestions!